bot/cogs/help_channels.py

```diff
@@ -713,6 +717,32 @@ async def on_message(self, message: discord.Message) -> None:
         # be put in the queue.
         await self.move_to_available()
 
+    @commands.Cog.listener()
+    async def on_message_delete(self, msg: discord.Message) -> None:
+        """Reschedule dormant when help channel is empty."""
```

Comment:
The documentation should be clearer.
```suggestion
        """
        Reschedule an in-use channel to become dormant sooner if the channel is empty.

        The new time for the dormant task is configured with `HelpChannels.deleted_idle_minutes`. 
        """
```